Code/Python/check pangram.py

```diff
@@ -0,0 +1,46 @@
+#Program to check if a given string is pangram or not
+
+#A pangram is a sentence containing every letter in the English Alphabet.
+
+
+#importing the string module
+from string import *
+
+#Function to check the string
+def pangram(string):
+    
+    
+    for alpha in ascii_lowercase:
+        
+        #checking for all the alphabets in the input string
+        #converting all the letters in lowercase
+        
+        if alpha not in string.lower():   
+            return False
+    
+    
+    return True
+
+
+#main
+string=input("Enter any string: ")
+
+
+#Function call
+if(pangram(string)==True):
+    
+    print("\n The given string is a pangram")
+          
+else:
+    print("\n The given string is not a pangram")
+
+
+    '''Test cases: 
+1) I/P:The Quick Brown Fox Jumps Over The Lazy Dog
+   O/P:The given string is a pangram
+
+2) I/P:2brown5! (including special characters and digits)
+   O/P:The given string is not a pangram
+'''
+
+#Time complexity:O(n)
```

Comment:
1. Include this in multline comment
2. Write space complexity too
3. specify n

Reply:
done!